pkg/meta/model/job_args.go

```diff
@@ -1404,63 +1181,50 @@ type PlacementPolicyArgs struct {
 	PolicyID int64 `json:"policy_id"`
 }
 
-func (a *PlacementPolicyArgs) fillJobV1(job *Job) {
+func (a *PlacementPolicyArgs) getArgsV1(job *Job) []any {
 	if job.Type == ActionCreatePlacementPolicy {
-		job.Args = []any{a.Policy, a.ReplaceOnExist}
+		return []any{a.Policy, a.ReplaceOnExist}
 	} else if job.Type == ActionAlterPlacementPolicy {
-		job.Args = []any{a.Policy}
-	} else {
-		intest.Assert(job.Type == ActionDropPlacementPolicy, "Invalid job type for PlacementPolicyArgs")
-		job.Args = []any{a.PolicyName}
+		return []any{a.Policy}
 	}
+	intest.Assert(job.Type == ActionDropPlacementPolicy, "Invalid job type for PlacementPolicyArgs")
```

Comment:
How about returning the error instead? If `job.Type` is not one of `ActionCreatePlacementPolicy`, `ActionAlterPlacementPolicy` or `ActionDropPlacementPolicy`, it should return an error in non-test environment.

Reply:
we can remove it actually, only 3 job types share this `Args` struct